__tests__/server/user-routes.spec.js

```diff
@@ -24,99 +25,53 @@ describe('Checks user answered routes', () => {
     expect(questionsRes.status).toBe(200)
     questions = questionsRes.body
     questionIDs = questions.map(question => question.ID)
-    
   })
   // the following test might modify `answered` field of user test2
   beforeEach(async () => {
-    clearFieldsInSingleRecord(userBaseName, user2Id, answeredFieldName)
-
+    await testSession.post('/api/auth/login').send({ Email: 'a2@b.com', Password: 'password'})

```

Comment:
on second thought, can you move this back to individual tests like you did with user test - I forgot that you have to do 2 separate logins